lib/mastodon/cli_helper.rb

```diff
@@ -7,3 +7,48 @@
 ActiveJob::Base.logger       = dev_null
 HttpLog.configuration.logger = dev_null
 Paperclip.options[:log]      = false
+
+module Mastodon
+  module CLIHelper
+    def create_progress_bar(total = nil)
+      ProgressBar.create(total: total, format: '%c/%u |%b%i| %e')
```

Comment:
I think we better add an explicit dependency on ruby-progressbar in the `Gemfile`?